moc_openshift.py

```diff
@@ -575,3 +619,60 @@ def update_rolebindings(self, project_name, role, rolebindings_json):
                 payload["metadata"][key] = rolebindings_json["metadata"][key]
         self.logger.debug("payload -> 2: " + json.dumps(payload))
         return self.put_request(url, payload, True)
+
+    # member functions for quotas
+    def generate_openshift_quota(self, project_name, quota_def):
+        payload = {
+            "apiVersion": "v1",
+            "kind": "ResourceQuota",
+            "metadata": {
+                "name": self.generate_quota_name(self, project_name, quota_def),
+                "namespace": project_name,
+            },
+            "spec": {},
+        }
+        for k in quota_def.keys():
+            if k == "scope":
+                payload["spec"]["scopes"] = [quota_def["scope"]]
+            else:
+                payload["spec"][k] = quota_def[k]
```

Comment:
You could replace the above with:

```
for k, v in quota_def.items():
  payload['spec']['scopes' if k == 'scope' else k] = v
```

Or even use a dictionary comprehension:

```
payload['spec'] = {'scopes' if k == 'scope' else k: v
                  for k, v in quota_def.items()}
```